Gems/PhysX/Code/Source/Common/PhysXSceneQueryHelpers.cpp

```diff
@@ -278,12 +314,25 @@ namespace PhysX
             return physx::PxQueryHitType::eNONE;
         }
 
+        // @deprecated Unused, we're only pre-filtering at this time
```

Comment:
update: Same comment before about the deprecation notice (could be the same one for all related code to make things easier to track)

Reply:
Removed this comment since we can't actually delete/deprecate this function until the base class function is removed from PhysX SDK